### DISCUSSION TITLE
How do I get my whole Real-debrid library in plex?

### DISCUSSION THREAD

**Discussion by GSKang94** [I-am-PUID-0/DMB#16] [Q&A] [answered]
Hi I've a noob question
I followed instructions and everything is running well but I can only see movies from my symlinks folder. My symlinks folder only contain links to newly added movies via Riven. I can see that rest of my Real-debrid library is still in `/DMB/Zurg/mnt` folder so should I also add that one to my plex library? If I do that then I'll have duplicates. 🤷🏼‍♂️

**Comment by I-am-PUID-0** [comment, marked as answer]
I don't believe Riven will process the existing content in your RealDebrid library; you'd need to re-add it by queuing it through Riven.
